### PR TITLE
fix/echam switches and version checks

### LINKED ISSUE #670: ECHAM6-wiso in standalone mode: wrong file directories for wiso output and restart files
When I run ECHAM6-wiso in standalone mode (and not as part of AWIESM-wiso), all *wiso* output and restart files are stored in the directory <EXP>/unknown instead of the directories <EXP>/output/[..] and <EXP>/restart/[..]. Therefore a simulation stops after the first job cycle as ECHAM6-wiso can't find the required wiso restart files in the expected directories.

To reproduce the bug, simply run echam6-wiso with namelist entries:

    add_namelist_changes:
            namelist.echam:
                wisoctl:
                    lwiso: True
                    lwiso_rerun: ${lresume}
                    nwiso: 3

(lresume is the default ECHAM6 switch for a cold start or restart.)

In the "move output" phase, the following files should be moved from <EXP>/run_xxxx/work:

to ECHAM output files:
<EXP>_XXX.01_wiso and <EXP>_XXX.01_wiso.codes
<EXP>_XXX.01_accw_wiso and <EXP>_XXX.01_accw_wiso.codes

to JSBACH output files:
<EXP>_XXX.01_js_wiso and <EXP>_XXX.01_js_wiso.codes
<EXP>_XXX.01_la_wiso and <EXP>_XXX.01_la_wiso.codes

to ECHAM restart files:
restart_<EXP>_<XXX>_wiso.nc 
restart_<EXP>_<XXX>_accw_wiso.nc 

to JSBACH restart files:
restart_<EXP>_<XXX>_js_wiso.nc 
restart_<EXP>_<XXX>_sf_wiso.nc 

This bug occurred on ollie.awi.de using esm_tools version 6.6.8.

### PR DESCRIPTION
**Pull request by mandresm**
Closes #670 and #671

### CODE REVIEW

File: src/esm_parser/esm_parser.py [hunk 1]
```diff
@@ -3022,6 +3024,43 @@ def _config_init(self, user_config):
...
+
+        Parameters
+        ----------
+        user_config : dict
+            Experiment configuration defined by the user (e.g. runscript)
+        setup_config : dict
+            Experiment configuration defined by the default ESM-Tools configuration
+            files (``<PATH>/esm_tools/configs/``)
+
+        Raises
```
Comment (denizural) [inline src/esm_parser/esm_parser.py]: Maybe not so important but `Raises` is used when the code explicity raises an Exception via the `raise` statement.

**Review by denizural — APPROVED** [review]
👍

File: src/esm_parser/esm_parser.py (hunk 1 above)
Comment (mandresm) [reply src/esm_parser/esm_parser.py]: Yeah, I was aware of that, however, I did not know where to put it otherwise. Do you have a suggestion?

Comment (denizural) [reply src/esm_parser/esm_parser.py]: NumPy docstring has an optional `Notes` section: https://numpydoc.readthedocs.io/en/latest/example.html#example
Maybe that could be the place.

Comment (pgierz) [reply src/esm_parser/esm_parser.py]: +1 for notes

**Review by pgierz — APPROVED** [review]

**Comment by pgierz** [comment]
Can I briefly give a giant high five regarding the new tests? I love seeing the little green check marks next to the commits :-)

**Comment by mandresm** [comment]
They are not yet here ahah, for this branch I am still doing them manually (there are still somethings to solve) but if you want to get excited you can look at the esm_tests/ollie and esm_tests/levante in #684 . There ollie pass all tests, but levante still has some problems. I hope I can merge that next week.

**Comment by mandresm** [comment]
After all tests passed and all issues addressed let's #bump!

**Pull request merged**